frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -181,6 +181,7 @@ function BookInfo:show(file, book_props)
 
     local widget = KeyValuePage:new{
         title = _("Book information"),
+        half_overflow_right_align = true,
```

Comment:
What about value_overflow_align

half_overflow ← you proposed custom space division

value_overflow ← in case we want to add this to key in the future

align ← left, (center), right

Because right_align = false? Does that mean center, justified, left? :-)

Reply:
Yes, makes sense.